validation/utils/compare_parquet_csv.py

```diff
@@ -13,6 +13,8 @@
 parser.add_argument('-c', '--csv-path', help='path to csv file', type=str, required=True)
 parser.add_argument('-s', '--sort', help='sort by string, int and categorical columns before check', action='store_true')
 parser.add_argument('--check-exact', help='Whether to compare number exactly.', action='store_true')
+parser.add_argument('--lower-case-cols', help='Lower cased column names before compare', action='store_true')
+parser.add_argument('--rasie-error', help='Lower cased column names before compare', action='store_true')
```

Comment:
there is a typo in rasie.
For help something like:
raise error if data are different